deploy/kubernetes/metricbeat-kubernetes.yaml

```diff
@@ -19,6 +19,9 @@ data:
         - type: kubernetes
           scope: cluster
           node: ${NODE_NAME}
+          # In large Kubernetes clusters consider setting unique to false
+          # to avoid using the leader election strategy and
+          # instead run a dedicated Metricbeat instance using a Deployment in addition to the DaemonSet
```

Comment:
Maybe point to https://www.elastic.co/guide/en/beats/metricbeat/current/running-on-kubernetes.html#_deploying_metricbeat_to_collect_cluster_level_metrics_in_large_clusters too?

Reply:
I tried that but the link is too long and it didn't look nice